src/snowflake/snowpark/_internal/utils.py

```diff
@@ -395,7 +395,10 @@ def parse_positional_args_to_list(*inputs: Any) -> List:
 
 
 def _hash_file(
-    hash_algo: hashlib._hashlib.HASH, path: str, chunk_size: int, whole_file_hash: bool
+    hash_algo: "hashlib._hashlib.HASH",
```

Comment:
This should be `"hashlib._Hash"`.

Reply:
emmm I didn't see _Hash defined in `hashlib`?

```python
>>> import hashlib
>>> hashlib._HASH
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'hashlib' has no attribute '_HASH'
>>> hashlib._hashlib.HASH
<class '_hashlib.HASH'>
>>> 

```